README.md

```diff
@@ -93,9 +93,53 @@ This also works with IPv6 CIDR ranges, for example:
 
 ```
 $ cidr overlaps 2001:db8:1111:2222:1::/80 2001:db8:1111:2222:1:1::/96
-true
+
 ```
 
+### CIDR division
+
+To divide a cidr range into distinct N distinct networks
+## IPV4
+```
+$ cidr divide 10.0.0.0/16 9
+  [Networks]
+10.0.0.0/20
+10.0.16.0/20
+10.0.32.0/20
+10.0.48.0/20
+10.0.64.0/20
+10.0.80.0/20
+10.0.96.0/20
+10.0.112.0/20
+10.0.128.0/20
+```
+
+## IPV6
+```
+$ cidr divide 2001:db8:1111:2222:1::/80 9
+  [Networks]
+2001:db8:1111:2222:1::/84
+2001:db8:1111:2222:1:1000::/84
+2001:db8:1111:2222:1:2000::/84
+2001:db8:1111:2222:1:3000::/84
+2001:db8:1111:2222:1:4000::/84
+2001:db8:1111:2222:1:5000::/84
+2001:db8:1111:2222:1:6000::/84
+2001:db8:1111:2222:1:7000::/84
+2001:db8:1111:2222:1:8000::/84
+
+```
+
+You can also use the -u flag to divide the network by desired users/hosts on your network. It assumes a Broadcast and Gateway address in the calculation. So you only have to think of hosts.
```

Comment:
```suggestion
You can also use the `-u` flag to divide the network based on the number of desired host addresses. This calculation includes a broadcast and gateway address, so you only need to consider the host addresses.
```

Reply:
I also suggest we change from the `-u` flag to something more related to "host addresses" since that's a more commonly accepted term. I can't quickly think of a good name for the flag, allow me to get back on this please.

Reply:
Oh I agree. My original desire was -h and --hosts. But cobra's default and the other standard is -h for help.
-d for domains?
-n for numbered hosts?
-c for clients (which is probably more technically accurate than users)?
but -u was for users was just the fastest sensible alternative when I realized -h failed.